core/inbox_test.go

```diff
@@ -5,12 +5,9 @@ import (
 	"fmt"
```

Comment:
This looks like it goes beyond the scope of removing `BlockHeight()`, in the future please do this in a separate commit at a minimum -- separate PR is fine too.